### PR TITLE
Transcript View: Update Interaction and Configurability

### PR DESCRIPTION
**Pull request by jbuckner**
**Description**

> This updates some of the user interactions with the transcript view as well as adds several styling variables so the consumer can style it better.

**Technical**

> n/a

**Testing**

> Run `yarn install`, `yarn test` to run tests. Run `yarn start` to play with it in the browser

**Evidence**

> n/a

### CODE REVIEW

File: packages/transcript-view/test/transcript-config.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {
+  html, fixture, expect, oneEvent
+} from '@open-wc/testing';
+
+import TranscriptEntryConfig from '../lib/src/models/transcript-entry-config';
+import TranscriptConfig from '../lib/src/models/transcript-config';
+
+describe('TranscriptConfig', () => {
+  it('properly filters searchRestults', async () => {
```
Comment (bfalling) [inline packages/transcript-view/test/transcript-config.test.js]: SUGGESTION: Fix typo `searchRestults`.

File: packages/transcript-view/test/transcript-entry.test.js [hunk 2]
```diff
@@ -58,11 +54,11 @@ describe('TranscriptEntry', () => {
       <transcript-entry .entry=${entry}></transcript-entry>
     `);
 
-    const clickableSpan = el.shadowRoot.querySelector('span');
+    // const clickableSpan = el.shadowRoot.querySelector('span');
```
Comment (bfalling) [inline packages/transcript-view/test/transcript-entry.test.js]: SUGGESTION: Remove this debug code?

File: packages/transcript-view/test/transcript-view.test.js [hunk 3]
```diff
@@ -16,6 +16,41 @@ describe('TranscriptView', () => {
     expect(el.config).to.equal(undefined);
   });
 
+  it('displays context zones if enabled', async () => {
+    const el = await fixture(html`
+      <transcript-view
+        showContextZones='true'>
+      </transcript-view>
```
Comment (bfalling) [inline packages/transcript-view/test/transcript-view.test.js]: QUESTION: You okay with the `<transcript-view>` start and end tags containing a newline? (Same question applies below.)

Comment (jbuckner) [reply packages/transcript-view/test/transcript-view.test.js]: @bfalling Are you talking about the parameters being on separate lines or the opening and closing tags being on separate lines or something else?

I mostly did it for readability.. once you start passing a few parameters into the component, it starts to get pretty dense so the line breaks just helps visually distinguish everything.

**Pull request merged**